solr/solr-ref-guide/build.gradle

```diff
@@ -184,6 +184,7 @@ ext {
         group "Documentation"
         description "Builds the ${details.desc}"
 
+        outputs.cacheIf { true }
```

Comment:
I'm not sure what it does here?

Reply:
it makes the task cacheable, same as putting `@Cacheable` annotation on a task class (not possible here since it's created on the fly)